app/assets/stylesheets/application.scss

```diff
@@ -18,3 +18,11 @@
  body {
    margin: 0px;
  }
+
+.navbar-right {
+  margin-right: 8px;
+}
+
+.flash {
+  margin-bottom: 0px;
+}
```

Comment:
For the navbar-right css: yes. The rest is applicable for the whole app, so I'll leave that in here.